spec/rubocop/cop/lint/format_parameter_mismatch_spec.rb

```diff
@@ -197,10 +197,6 @@
     expect_no_offenses('format("%0*x", max_width, id)')
   end
 
-  it 'accepts an extra arg for dynamic width with other following flags' do
-    expect_no_offenses('format("%*0x", max_width, id)')
```

Comment:
It turned out that `%*0x` is not a valid format sequence. MRI does not accept it.